src/notifications/components/NotificationList.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import styles from './NotificationList.css'
+
+const NotificationList = ({ children }) => (
+    <ul className={styles.root}>{children}</ul>
```

Comment:
Some of the `props.children` are going to be `<div>` els rather than `<li>`s (`<StatusHeading>` and `<ReadHeader>` components). If this works for now, I would leave it but we should think about restructuring this overall view as two `<ul>`s for read and unread lists